### ISSUE TITLE
Wrong autocompletion for Carbon::createFromFormat return value type

### ISSUE THREAD

**Issue by Kozubsky** [briannesbitt/Carbon#1857]
Hello,

I encountered an issue with the following code:
```php
$normalizedDateTime = Carbon::createFromFormat('Y-m-d H:i:s', $datetime, 'UTC');
```

Carbon version: 2.23.1

PHP version: 7.2.19-0ubuntu0.18.04.2

I expected to get:

```
Autocompletion in PhpStorm for $normalizedDateTime as Carbon instance
```

But I actually get:

```
Autocompletion in PhpStorm for $normalizedDateTime as DateTime instance.
So, to fix this issue, I suggest to add this line to PHPDoc section of Carbon/Carbon class:
 * @method static Carbon         createFromFormat($format, $time, $tz = null)
```

Thanks!

**Comment by kylekatarnls** [comment]
Hi,

The PHPDoc is properly set in `CarbonInterface` and the trait `Creator` we should not add one more `@method` which should be used only for magic methods.

I need first to check why PHPStorm fallback to the `DateTime` doc skipping the `Carbon`'s one.

**Comment by kylekatarnls** [comment]
Will be released in *2.24.0*
